index.js

```diff
@@ -18,6 +18,7 @@ var couch2pg = require('couch2pg'),
     xmlforms = require('./libs/xmlforms/updater')(db);
 
 var firstRun = false;
+var authError = false;
```

Comment:
this var does not prevent `firstRun` from being set to true, which while it does not matter in its current state, is not technically correct and could cause new issues in the code as things are refactored.